virtualizarr/tests/test_parsers/test_kerchunk.py

```diff
@@ -150,7 +153,10 @@ def test_handle_relative_paths(refs_file_factory):
     refs_file = refs_file_factory(chunks={"a/0.0": ["test1.nc", 6144, 48]})
```

Comment:
the changes to `test_handle_relative_paths` are temporary if https://github.com/zarr-developers/VirtualiZarr/pull/662/ is also adopted